src/java.base/share/classes/java/util/concurrent/ForkJoinPool.java

```diff
@@ -983,7 +956,8 @@ public class ForkJoinPool extends AbstractExecutorService {
     static final long DEFAULT_KEEPALIVE = 60_000L;
 
     /**
-     * Undershoot tolerance for idle timeouts
+     * Undershoot tolerance for idle timeouts, also serving as the
+     * minimum allowed timeout value.
      */
     static final long TIMEOUT_SLOP = 20L;
```

Comment:
Might be worth spelling out the temporal unit used here.

Reply:
done.